--- mp4v2-1.9.1/src/mp4file.cpp~	2009-07-14 01:07:12.000000000 +0200
+++ mp4v2-1.9.1/src/mp4file.cpp	2012-11-13 14:19:21.505940224 +0100
@@ -2580,7 +2580,7 @@
     GetChapters(&chapters, &chapterCount, sourceType);
     if (0 == chapterCount)
     {
-        VERBOSE_READ(GetVerbosity(), printf(errMsg));
+        VERBOSE_READ(GetVerbosity(), printf("%s", errMsg));
         return MP4ChapterTypeNone;
     }
 
--- mp4v2-1.9.1/src/mp4property.cpp~	2009-07-14 01:07:11.000000000 +0200
+++ mp4v2-1.9.1/src/mp4property.cpp	2012-11-13 14:20:00.485938846 +0100
@@ -619,7 +619,7 @@
 
         oss << "  |" << text.str() << "|\n";
 
-        fprintf( pFile, oss.str().c_str() );
+        fprintf( pFile, "%s", oss.str().c_str() );
         fflush( pFile );
         return;
     }
