--- mp4v2-Release-MP4v2-3.0.4.0/libutil/Utility.h.orig	2019-04-08 15:28:47.000000000 +0200
+++ mp4v2-Release-MP4v2-3.0.4.0/libutil/Utility.h	2020-07-04 21:51:51.847120315 +0200
@@ -52,7 +52,7 @@
 {
 protected:
     enum LongCode {
-        LC_NONE = 0xf0000000, // safe (cannot conflict with char values)
+        LC_NONE = 0xf000000, // safe (cannot conflict with char values)
         LC_DEBUG,
         LC_VERBOSE,
         LC_HELP,
